src/server/game/Miscellaneous/SharedDefines.h

```diff
@@ -679,7 +679,7 @@ enum SpellAttr7 : uint32
     SPELL_ATTR7_SILENCE_ONLY_NONPLAYER           = 0x00001000, // TITLE Can Cause Silence
     SPELL_ATTR7_CAN_ALWAYS_BE_INTERRUPTED        = 0x00002000, // TITLE No UI Not Interruptible DESCRIPTION Can always be interrupted, even if caster is immune
     SPELL_ATTR7_UNK14                            = 0x00004000, // TITLE Unknown attribute 14@Attr7
-    SPELL_ATTR7_UNK15                            = 0x00008000, // TITLE Unknown attribute 15@Attr7 DESCRIPTION Exorcism - guaranteed crit vs families?
```

Comment:
the comment on this line needs a certain syntax: `TITLE <title> DESCRIPTION <description>` which will then be parsed by https://github.com/TrinityCore/TrinityCore/blob/master/contrib/enumutils_describe.py to generate the file `enuminfo_SharedDefines.cpp`

Reply:
@mdX7 doesnt matter - SharedDefines changes wont be merged (i plan to tear apart this PR and push it in parts)